one-pager/banner/main.py

```
@@ -0,0 +1,71 @@
+import functions_framework
+import json
+import uuid
+from PIL import Image
```

Ruff (0.12.2)

`PIL.Image` imported but unused

Remove unused import: `PIL.Image`

(F401)